app/views/quotes/index.html.erb

```diff
@@ -9,7 +9,11 @@
       </footer>
     </blockquote>
     <div class="created">
+    <% if quote.created_at %>
       <%= time_ago_in_words(quote.created_at) %> ago
+    <% else %>
+      <%= time_ago_in_words(Time.new) %> ago
+      <% end %>
```

Comment:
Great we'll merge this right away to avoid any further production issues. That said, this is just a workaround. can you open another PR with code that fixes the data in our production.sql file as well as uses constraints and validators to make sure we *never* have a null `created_at` field again? thanks.